frontend/public/components/factory/table-filters.ts

```diff
@@ -29,7 +29,7 @@ export const fuzzyCaseInsensitive = (a: string, b: string): boolean =>
 
 // TODO: Table filters are undocumented, stringly-typed, and non-obvious. We can change that.
 export const tableFilters: TableFilterMap = {
-  name: (filter, obj) => fuzzyCaseInsensitive(filter, obj.metadata.name),
+  name: (filter, obj) => fuzzyCaseInsensitive(filter, obj.metadata?.name || obj.path),
```

Comment:
The data source for this list is a not a K8sResource, hence this is failing the filter by name